schulcloud/FilesViewController.swift

```diff
@@ -8,10 +8,12 @@
 
 import UIKit
 import CoreData
+import BrightFutures
 
 class FilesViewController: UITableViewController, NSFetchedResultsControllerDelegate {
 
     var currentFolder: File!
+    var fileSync = FileSync()
```

Comment:
I'm creating `URLSession`s for each controller, I use that to be able to cancel tasks when this object is destroyed